.github/workflows/app-react.yml

```diff
@@ -47,3 +57,17 @@ jobs:
           env_vars: javascript
           # Specify whether or not CI build should fail if Codecov runs into an error during upload
           fail_ci_if_error: true
+
+      - name: SonarQube Scan
+        uses: sonarsource/sonarqube-scan-action@master
+        if: ${{ github.event_name == 'push' }}
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          SONAR_HOST_URL: ${{ secrets.SONAR_URL }}
+          PROJECT_KEY: 21faa627-fe70-4a67-89e3-1262a32d2e2c
```

Comment:
This is fine for now but maybe we should have that as a secret?

Reply:
It's technically not a secret.